jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -54,6 +56,19 @@ def __init__(self, *args, **kwargs):
                                           'subset_items',
                                           'subset_selected',
                                           default_text="Create New")
+        self.all_subsets_with_subregions = {}
+        self.all_subset_types = {}
+
+    @property
+    def user_api(self):
+        return PluginUserApi(self, expose=('subset_select', 'subset_definitions',
+                                           'get_all_subsets_with_subregions'))
```

Comment:
We should not expose this yet because this does not work for *all* the subset types yet. You can still access the API by doing `plugin_obj._obj.get_all_subsets_with_subregions(...)` so you don't really need this.

Reply:
This is in 1st draft mode, I plan  on adding coverage for other subset types if this 1st draft seems fine.